plugins/scaffolder-backend-module-cookiecutter/api-report.md

```diff
@@ -0,0 +1,22 @@
+## API Report File for "@backstage/plugin-scaffolder-backend-module-cookiecutter"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ContainerRunner } from '@backstage/backend-common';
+import { ScmIntegrations } from '@backstage/integration';
+import { TemplateAction } from '@backstage/plugin-scaffolder-backend';
+import { UrlReader } from '@backstage/backend-common';
+
+// @public (undocumented)
+export function createFetchCookiecutterAction(options: {
+    reader: UrlReader;
+    integrations: ScmIntegrations;
+    containerRunner: ContainerRunner;
```

Comment:
i wonder if it would actually be a nicety in the interface to make the container runner optional, and throw an error if we don't find a local version 🤔